node/builder_chain.go

```diff
@@ -48,6 +48,13 @@ var ChainNode = Options(
 	// Full node or lite node
 	// TODO: Fix offline mode
 
+	// FVM: builtin actor bundle loading
+	//      Note: this has to load before the upgrade schedule, so that we can patch in the
+	//      right manifest cid.
+	//      This restriction will be lifted once we have the final actors v8 bundle and we know
+	//      the manifest cid.
+	Override(BuiltinActorsKey, modules.LoadBultinActors),
```

Comment:
Does this get called? If it's not using an InvokeKey, it's just a Provide, and given that this isn't really a constructor, I wouldn't expect it to get called.

On invoke keys, note that they are called in order the invoke keys are defined, and are essentially the 'roots' from which dependencies get resolved.

Probably the best (a.k.a least bad) way to do that is to make `LoadBultinActors` return a dummy type like `type BuiltinActorsLoaded struct{}` and depend on it in UpgradeSchedule constructor (we do a similar thing when loading genesis into the chainstore)

Reply:
ah thanks, will do the dummy type thing.

Reply:
done.